### DISCUSSION TITLE
E' possibile utilizzare le opzioni di customizzazione della fattura elettronica per una fattura normale ?

### DISCUSSION THREAD

**Discussion by pupiweb** [fattureincloud/api#304] [Q&A] [answered]
Buongiorno -- 

Ho un cliente che desidererebbe passare in fattura i dati relativi a contratti e DDT delle righe della fattura, analogamente a quanto si fa con la fattura elettronica

E' possibile utilizzare le opzioni di E-Invoice XML customization in una fattura "normale" ?

Passare cioè nel payload una serie di costrutti tipo
    "ei_raw": {
          "FatturaElettronicaBody": {
              "DatiGenerali": {
                  "DatiRicezione": [
                      {
                          "NumItem": "5"

Grazie ;)

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
chiedo scusa, ma qual è il risultato finale che si vorrebbe ottenere?

Il campo ei_raw ha senso solo nell'ambito della fatturazione elettronica, in quanto permette di definire i parametri indicati fanno parte dello schema indicato dall'agenzia delle entrate per le fatture elettroniche.

Grazie
Mauro

**Comment by pupiweb** [comment]
> Buongiorno,
> chiedo scusa, ma qual è il risultato finale che si vorrebbe ottenere?
> 
> Il campo ei_raw ha senso solo nell'ambito della fatturazione elettronica, in quanto permette di definire i parametri indicati fanno parte dello schema indicato dall'agenzia delle entrate per le fatture elettroniche.

E' quello che mi sembrava di aver capito, ma spinto dall'insistenza del cliente  ho preferito chiedere, nel dubbio di essermi sbagliato

grazie per la risposta, 

g. pupita

**Reply by valmoz** [reply]
Di niente :) 

Mauro
